GitHubDashboard/ClientApp/app/components/query/count.component.ts

```diff
@@ -51,5 +55,16 @@ class QueryCountResult {
                 this.query += `label%3A${label} `;
             }
         }
+
+        if (excludedMilestone)
+        {
+            this.query = `-milestone%3A"${excludedMilestone}" `;
+        }
+
+        if (excludedLabels) {
+            for (let label of excludedLabels.toString().split(",")) {
+                this.query += `-label%3A${label} `;
```

Comment:
Consider - quoting this as you did milestones above.

Reply:
Fixed it and also updated the original implementation for plain labels

---
In reply to: [247577868](https://github.com/Pilchie/GitHubDashboard/pull/13#discussion_r247577868) [](ancestors = 247577868)